github_pr.py

```diff
@@ -75,6 +76,81 @@ def _load_issue(**args):
     return repo.get_issue(args['number'])
 
 
+def __validate_list_of_dict(list_of_dict):
```

Comment:
you should pretty much never use double underscore. This is a better description: http://docs.python-guide.org/en/latest/writing/style/#we-are-all-responsible-users